src/OpenTK.Windowing.Common/Interfaces/INativeWindow.cs

```diff
@@ -237,7 +237,8 @@ public interface INativeWindow : IDisposable
         /// <summary>
         /// Makes the GraphicsContext current on the calling thread.
         /// </summary>
-        void MakeCurrent();
+        /// <param name="makeCurrent">If the GraphicsContext should be current on the calling thread. Set to false to make it so that <b>no</b> GraphicsContext is current on the calling thread.</param>
+        void MakeCurrent(bool makeCurrent);
```

Comment:
A method named `MakeCurrent` that does not in fact make the context current? This definitely could use a different name. It's very non-obvious what calling this with false does -  can we remove the parameter and add a second method called `DetachContext()` ?

Reply:
This is fixed now, not by introducing another function but by exposing a `IGraphicsContext` in `INativeWindow`.
We might even consider removing the `MakeCurrent` method as it is no longer needed, but it might be a nice convenience.